src/Microsoft.IdentityModel.Abstractions/ITelemetryClient.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Interface for Telemetry tracking.
+    /// </summary>
+    public interface ITelemetryClient
```

Comment:
Are there tests?

Reply:
What sort of interface tests would you like to see? There will certainly be tests for the implementation but there's no code here.

Reply:
Added some tests to ensure the contract isn't broken to add friction to breaking the public contract.